modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js

```diff
@@ -474,7 +474,7 @@ function convertMaterial(sourceMaterial) {
     };
   } else if (sourceMaterial.emissiveTexture) {
     texture = sourceMaterial.emissiveTexture.texture.source;
-    material.emissiveTexture = {
+    material.pbrMetallicRoughness.baseColorTexture = {
```

Comment:
Worth a comment explaining why we do this?

Reply:
I don't know why. Arcgis doesn't work with `emissiveTexture`, but work with `baseColorTexture`

Reply:
In that case I would definitely put that into a comment. If not, how is the next maintainer know that?

@Tamrat-B